Repubbies Pubbystation

pubby opened the pull request:
Here's an update for my map just to keep it fresh.

### Some bugfixes:

- Fixed hallway airlocks that were previously named "Diner" (they're now unnamed).
- Fixed improper areas in gateway room and xenobiology.
- Fixed bad camera tags.
- Fixed APC in engineering that was too close to the singulo.
- Fixed monastery shuttle lights floating in space.
- Added full glass windows to the emergency shuttle so that shuttle rotation code bugs stop letting all the air leak out. 

### Some requested changes:
- There's now a transit tube to the monastery, in addition to the shuttle.
- The turbine engine was moved from science to atmospherics.
- Added a deep fryer to the kitchen.

### Some wacky experimental redesigns:
- Medbay and science are now public hallways. This is to make the hallway topology more interesting to wander, and also to make lowpop have more interaction. At least, that's the idea.
- Genetics now has a fucking monkey sanctuary.
- Aux base is now in maintenance and it's fucking huge.

### Other changes:
- Most departments now have blue signs labeling them.
- Colored lights added.
- The bar can now be locked down with the push of a button.
- Viro and Xenobiology have windows looking into maint so that you can spy on them.

### New station map

[It's full of screenshot artifacts.](https://s4.postimg.org/9dsm5s323/stitched-fs8.png)

### New sprites

![image](https://user-images.githubusercontent.com/1446540/28694388-e0817754-72ee-11e7-8160-8f1d7d734081.png)

:cl: Pubby
experiment: PubbyStation's layout is now more public.
add: PubbyStation's turbine engine is now in atmospherics.
add: PubbyStation's monastery can now be reached by transit tube.
tweak: PubbyStation's aux mining base is now in engineering maint.
/:cl:

Repukan commented:
Spicy

Supermichael777 commented:
we already have arrow signs

ShizCalev commented:
![3munj3t](https://user-images.githubusercontent.com/6209658/28696794-eca26c34-7305-11e7-81ba-69a898fe087d.png)
South wall of the arrivals lounge has the wrong area.

![3munlvi](https://user-images.githubusercontent.com/6209658/28696876-59d08020-7306-11e7-979a-a7ca6b209595.png)
Engie storage area doesn't fill the whole area.

![image](https://user-images.githubusercontent.com/6209658/28696916-92af61ae-7306-11e7-8489-809391070d1a.png)
random maint area on some windows outside atmos

![3munqyg](https://user-images.githubusercontent.com/6209658/28697056-69ce00be-7307-11e7-89b0-68b58955092c.png)
viro's area is incorrect

![image](https://user-images.githubusercontent.com/6209658/28697247-7421b30c-7308-11e7-8c6d-519bced371a0.png)
pipes switch types in the morgue (under some cables)

![image](https://user-images.githubusercontent.com/6209658/28697303-d3ee50ec-7308-11e7-9cf8-943489edc2f2.png)
unconnected cable under the door in the transit room

![image](https://user-images.githubusercontent.com/6209658/28697323-f8a17536-7308-11e7-898a-0695d5dce9d4.png)
same under the port solar door as well

![image](https://user-images.githubusercontent.com/6209658/28697428-dc165d5e-7309-11e7-90ec-de03873cbb44.png)
prison door

![image](https://user-images.githubusercontent.com/6209658/28697355-3df827ba-7309-11e7-9657-30cde80d4021.png)
cable leading to nowhere above toxins

Would recommend scooting the Northern air vent in the nuke room over a tile since there will be a layer popping issue with the terminal right above it

ShizCalev commented on _maps/shuttles/emergency_pubby.dmm:
```diff
@@ -1,629 +1,77 @@
-//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
```
mapmerge :(

pubby commented:
@ShizCalev Thanks. That's really helpful.

And I'm incompetent as FUCK right now.

pubby committed:
Updates PubbyStation

- makes layout more public
- adds monastery transit tube
- aux mining base is now in engineering maint.
- and more!

pubby commented:
I force-pushed an update to replace the previous mess with a single commit.

Tacolizard commented:
pubby has always been the best map, and this reaffirms my stance

octareenroon91 commented:
Can high-pop miners get to the aux base? I don't remember them getting maint access except in lowpop.

pubby commented:
>Can high-pop miners get to the aux base?

I don't think high pop miners can enter maint but I don't consider it much of an issue.

>Cryo is public-access but sleepers are not?

Yep. It takes 8 years of medical school to know how to open the sleeper room airlock :)

(the medbay layout is kinda an experiment)

GunHog commented:
Why is Cargo not public access as well?

pubby commented:
My theory was that public access departments would only work if they doubled as public access hallways, as that would bring people into the department who would not normally go there. For example, players traveling from the brig to engineering can now cut through medbay, which hopefully leads to more player interaction.

Cargo on the other hand is one big dead-end, and so it can't really function as a hallway. The only reason people would ever go to public-access cargo is if they wanted to do a cargo tech's job for him, and I think that reduces interaction rather than adds it.

PromethTK11 commented:
Changing the singularity for a super matter when

PraiseRatvar commented:
i prefer if pubby was still married to the singularity engine since it makes the map more unique
